modules/client/src/tokenVoting/internal/graphql-queries/token.ts

```diff
@@ -1,8 +1,8 @@
 import { gql } from "graphql-request";
 
-export const QueryToken = gql`
+export const QueryTokenVotingToken = gql`
```

Comment:
This feels ambiguous. How about `QueryTokenVotingTargetContract` or something more specific?

Reply:
changed